runtime/Cpp/runtime/src/misc/MurmurHash.cpp

```diff
@@ -17,118 +20,83 @@ using namespace antlr4::misc;
 
 #if defined(_MSC_VER)
 
-#define FORCE_INLINE	__forceinline
```

Comment:
I assume FORCE_INLINE is no longer needed or even counter productive?

Reply:
Yeah, it was never referenced.